web/content/_index.md

```diff
@@ -6,7 +6,13 @@ This is work-in-progress documentation site for <a href="https://www.theforeman.
 
 Official documentation [is available here](https://theforeman.org/manuals/nightly/index.html).
 
-Available documentation sets are:
+Available upstream documentation sets are:
 
 * [Nightly](/nightly.html)
 * [2.3](/2.3.html)
+
+Other relevant documentation:
+
+* [Red Hat Satellite](https://access.redhat.com/documentation/en-us/red_hat_satellite) is a distribution of Foreman and several plugins including Katello, Remote Execution, Discovery, Bootdisk, OpenSCAP for the enterprise.
+* [ATIX orcharhino](https://docs.orcharhino.com/or/docs/index.html) is a distribution of Foreman and many plugins providing vendor-independent management of Red Hat, Suse, Oracle and Debian/Ubuntu nodes
+* [Red Hat Satellite Performance Tuning](https://access.redhat.com/solutions/1257143) is a set of performance-related resources, a free Red Hat portal account is required to access the content.
```

Comment:
```suggestion
* [Red Hat Satellite Performance Tuning](https://access.redhat.com/solutions/1257143) is a set of performance-related resources, a free Red Hat portal account is required to access the content. Note that while some of the guidance here might help you, Red Hat Satellite is a combination of Foreman and Katello. The latest Satellite release is always two versions behind the latest Foreman release
```

I would like some caveat like this added.

Reply:
:+1: for a caveat. However, currently 6.8 is based on 2.1 and 2.4 is stable so it's even 3 releases. Perhaps you can phrase it in a way that's less committed to the exact number of versions?

Reply:
A teetered between "several" and "a few". I thought "a few" was less dramatic. Updated ^